classes/class-wc-connect-jetpack.php

```diff
@@ -86,5 +86,26 @@ public static function build_connect_url( $redirect_url ) {
 				'woocommerce-services-auto-authorize'
 			);
 		}
+
+		/**
+		 * Records a Tracks event
+		 * @param $user
+		 * @param $event_type
+		 * @param
+		 */
+		public static function tracks_record_event( $user, $event_type, $data ) {
+			if ( ! class_exists( 'Jetpack' ) && defined( JETPACK__VERSION ) ) {
```

Comment:
Are you sure about this `&& defined( JETPACK__VERSION )` condition? What are you trying to test for here?